README.md

```diff
@@ -1,24 +1,78 @@
 # ICCS PyTorch training
 
 This repository contains documentation, resources, and code for the Introduction to
-Machine Learning with PyTorch session deliovered by @jatkinson1000 and @jdenholm of
+Machine Learning with PyTorch session deliovered by [**@jatkinson1000**](https://github.com/jatkinson1000)
+and [**@jdenholm**](https://github.com/jdenholm) of
 [ICCS](https://github.com/Cambridge-ICCS) at the [2023 summer school](https://iccs.cam.ac.uk/events/iccs-summer-school-2023).
 
 
 ## Contents
-[Installation and Setup](#installation-and-setup)
+
+- [Preparation and prerequisites](#preparation-and-prerequisites)
+- [Installation and setup](#installation-and-setup)
+- [Teaching material](#teaching-material)
+
+
+## Preparation and prerequisites
+
+To get the most out of the session we expact a basic understanding in a few areas and for you to do some preparation.
+Topics expected are outlined below, along with some resources for reading if you are unfamiliar.
+
+### Mathematics and Machine Learning
+
+Basic knowledge of:
+- calculus - differentiating a function
+- matrix algebra - matrix multiplication and representing data as a matrix
+- regression - fitting a function to data
+
+Neural Networks:
+- Awareness of high-level concepts 
+- We recommend the [video series by 3Blue1Brown](https://www.3blue1brown.com/topics/neural-networks), at least chapters 1-3.
+
+### Python
+The course will be taught in python using pyTorch.
+Whilst no prior knowledge of pyTorch is expected we assume users are familiar with the basics of Python3.
+This includes:
+- Basic mathematical operations
+- Writing and running scripts/programs
+- Writing and using functions
+- The conjcept of object orientation (that an object, e.g. a dataset, can have associated functions/methods associated with it.)
```

Comment:
Maybe add link (e.g. to wikipedia [OOP](https://en.wikipedia.org/wiki/Object-oriented_programming), or some more programmer-oriented website.